multiqc/modules/gfastats/gfastats.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+""" MultiQC module to parse output from gfastats """
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Import the gfastats submodules
+from .stats import StatsReportMixin
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule, StatsReportMixin):
+    """gfastats has a number of different commands and outputs.
+    This MultiQC module supports some but not all. The code for
+    each script is split into its own file and adds a section to
+    the module output if logs are found."""
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="gfastats",
+            anchor="gfastats",
+            target="gfastats",
+            href="https://github.com/vgl-hub/gfastats",
+            info=" is a single fast and exhaustive tool for summary statistics and simultaneous *fa* (fasta, fastq, gfa [.gz]) genome assembly file manipulation.",
+            doi="",
```

Comment:
Please don't set if there is no DOI. Use a comment instead.
```suggestion
            # doi="",  # Can't find a DOI
```